### ISSUE TITLE
Get "Terms of Use" from Collibra

### ISSUE THREAD

**Issue by aronhoyer** [equinor/data-marketplace#149]
- Discuss with Data Catalog team whether or not `restrictions` can be used in place of the domain `rights-to-use`

**Comment by aronhoyer** [comment]
after researching the collibra api docs, it doesn't seem as simple as i initially thought to get the asset restrictions. therefore, i think we're just gonna use the old approach i suggested

**Comment by aronhoyer** [comment]
depends on #133

**Comment by nilsml** [comment]
Direct checkout means 133 is no longer needed
